src/runtime/internal/cache.ts

```diff
@@ -349,6 +349,7 @@ export function defineCachedEventHandler<
           fetch: globalThis.$fetch as any,
         })) as $Fetch<unknown, NitroFetchRequest>;
       event.context = incomingEvent.context;
+      event.context.cached = true;
```

Comment:
Maybe `isCached`? 
```suggestion
      event.context.isCached = true;
```

Reply:
Makes more sense yes.

Also I should check if it respect the `shouldBypassCache` too